src/engine/specification.md

```diff
@@ -511,3 +540,30 @@ Refer to the specification for [`engine_getPayloadV1`](#engine_getpayloadv1).
 7. Considering the absence of the `TERMINAL_TOTAL_DIFFICULTY` value (i.e. when a value has not been decided), Consensus Layer and Execution Layer client software **MUST** use `115792089237316195423570985008687907853269984665640564039457584007913129638912` value (equal to`2**256-2**10`) for the `terminalTotalDifficulty` input parameter of this call.
 
 [json-rpc-spec]: https://playground.open-rpc.org/?schemaUrl=https://raw.githubusercontent.com/ethereum/execution-apis/assembled-spec/openrpc.json&uiSchema[appBar][ui:splitView]=false&uiSchema[appBar][ui:input]=false&uiSchema[appBar][ui:examplesDropdown]=false
+
+## Optional endpoints
+
+### engine_newPayloadHeaderV1
+
+#### Request
+
+* method: `engine_newPayloadHeaderV1`
+* params:
+  1. [`ExecutionPayloadHeaderV1`](#ExecutionPayloadHeaderV1)
+* timeout: 8s
+
+#### Response
+
+Refer to the response for [`engine_newPayloadV2`](#engine_newpayloadv2).
+
+#### Specification
+
+1. Execution Layer client software **MUST** handle calls to this endpoint the same as [`engine_newPayloadV2`](#engine_getpayloadv2) and provide compatible responses. When needed, Execution Layer client software **MUST** obtain the block's transactions and withdrawals autonomously.
```

Comment:
EL client can't run payload validation as part of processing this call. Strictly, the only possible status in the response to this call is `SYNCING`, because body of the payload is missing thus there is not enough data to run validation. The corresponding `forkchoiceUpdated` may result in validity payload status as EL may pull the body in between calls and run full validation on `forkchoiceUpdated`.

So, I would strictly specify the way EL handles this particular call.

Reply:
EL could do the syncing and validation as part of the call as long as it's within timeout.
Likewise, for duplicates it could return cached known `VALID/INVALID` answers.

Overall, agree that `newPayloadHeader` would return more `SYNCING` compared to `newPayload`, but not solely `SYNCING`.